### PR TITLE
VST lupa parser and public view

### PR DESCRIPTION
**Pull request by ghost**
Refactored the generic lupa view and parser to minimize duplicate implementation with corresponding VST view.
Implement parser for VST lupa data, according to kuja-template/lupahistoria/liikunnankoulutuskeskukset/paatos/content_paatos_fi

### CODE REVIEW

File: src/stores/pebbleLupa.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+
+const Store = createStore({
+  initialState: {},
+  actions: {
+    load: ({uuid}) => ({ getState, setState }) => {
+      return execute(
+        { getState, setState },
+        {
+          key: "pebbleLupa",
+          urlEnding: `${uuid}`
```
Comment (rapitkan) [inline src/stores/pebbleLupa.js]: Why is uuid in `${ }` ? Why not just urlEnding: uuid ?

File: src/utils/lupaParser.js [hunk 2]
```diff
@@ -1,35 +1,94 @@
...
+ * Return an object with lupamääräys data categorized into sections matching VST template in
+ * kuja-template/lupahistoria/liikunnankoulutuskeskukset/paatos
+ *
+ * Depends on VST_LUPA_STRUCTURE to guide the order and logic of parsing määräys data and
+ * producting formatted section data, based on kohde or koodisto identifier. Uses react-intl
+ * utilities for primary localization and that based on koodisto data.
+ *
+ * @param lupa lupa with määräys and järjestäjä as given by backend
+ * @param intl react-intl utility object
+ * @returns [{heading: string, values: string[]}]
```
Comment (rapitkan) [inline src/utils/lupaParser.js]: According to JSDoc the syntax of parameters is the following (e.g):

@param {string} somebody Somebody's name.

I'm not sure if parameter can be written without { }.

Syntax for @returns is (e.g.): @returns {number} Sum of a and b

File: src/utils/lupaParser.js [hunk 3]
```diff
@@ -1,35 +1,94 @@
...
+    return sectionDataList;
+  }
+};
+
+const generateOrganizerSectionData = (lupa, locale) => {
+  const kunta = parseLocalizedField(lupa.jarjestaja.kuntaKoodi.metadata, locale.toUpperCase())
+  // Exception sourced from kuja-template/lupahistoria/liikunnankoulutuskeskukset/paatos/content_paatos_fi.html:35
+  // TODO: localization of this exception case content
+  const value = lupa.jarjestaja.oid === '1.2.246.562.10.13451568789' ?
+    `${lupa.jarjestaja.nimi[locale]}, ${kunta} sekä Humppilan ja Ypäjän kunnat` :
```
Comment (rapitkan) [inline src/utils/lupaParser.js:55]: Swedish translation and localization in common not needed?

File: src/utils/lupaParser.js [hunk 4]
```diff
@@ -41,117 +100,189 @@ const generateIteratedKoodiData = (maaraykset, locale) => {
...
-    return generateErityinenKoulutustehtavaData(maarays, locale);
+
+  return data;
+};
+
+const generateOppilaitoksetDataForVST = (maaraykset, locale) => {
+  let values = [];
+
+  for(const maarays of maaraykset) {
+    console.log(maarays)
```
Comment (rapitkan) [inline src/utils/lupaParser.js]: Is this console.log important?

**Review by rapitkan — APPROVED** [review]

File: src/stores/pebbleLupa.js (hunk 1 above)
Comment (ghost) [reply src/stores/pebbleLupa.js]: Was copy and replace from lupa equivalent with ytunnus in the same place. Didn't think through. Anyway the file wasn't supposed to be added and was removed already.

File: src/utils/lupaParser.js (hunk 3 above)
Comment (ghost) [reply src/utils/lupaParser.js:55]: Same kind of hardcoded exception in the html template. Should be fixed separately and out of scope here.

File: src/utils/lupaParser.js (hunk 4 above)
Comment (ghost) [reply src/utils/lupaParser.js]: No it's a mistake.

**Pull request merged**